Makefile

```diff
@@ -1,6 +1,9 @@
 VERSION ?= 1.0.0
 REVISION ?= git-$(shell git rev-parse --short HEAD)
 
+k8s-aws-elb-tagger: cmd/k8s-aws-elb-tagger/main.go
+	go build -o k8s-aws-elb-tagger cmd/k8s-aws-elb-tagger/main.go
+
 k8s-aws-elb-tagger.linux.amd64: cmd/k8s-aws-elb-tagger/main.go
 	GOOS=linux GOARCH=amd64 go build -o k8s-aws-elb-tagger.linux.amd64 cmd/k8s-aws-elb-tagger/main.go
```

Comment:
For cross environment, it is preferable to output result in a dedicated folder and add it in the .gitignore file.